services/parser-database/tests/test_connector.py

```diff
@@ -92,3 +92,17 @@ def test_if_got_error_from_keywords_service(mock_get):
     mock_get.return_value.json.return_value = {'error': {'message': "something happened"}}  # noqa: E501
     with pytest.raises(Exception):
         assert connector.get_keywords(text_to_keywordize)
+
+
+def test_keywords_in_memory():
+    expected_in_memory = {"forzoso": [{"id": 1, "number": 1, "frequency": 1}],
+                          "bicicleta": [{"id": 1, "number": 1, "frequency": 1}],
+                          "usar": [{"id": 1, "number": 1, "frequency": 1}],
+                          "casco": [{"id": 1, "frequency": 1, "number": 1}]}
+    try:
+        connector.save_keywords_in_memory(
+            ["forzoso", "bicicleta", "usar", "casco"],
+            {"id": 1, "number": 1, "content": "en bicicleta es siempre forzoso usar casco"})
+        assert expected_in_memory == connector.keywords_in_memory
+    except Exception as e:
+        pytest.fail("Error: " + str(e))
```

Comment:
Good test! I only have one concern: because `connector.keywords_in_memory` is global state, it could be affected by other tests. The safest way to avoid this is to add
```
assert {} == connector.keywords_in_memory
```

at the *beginning* of the unit test. When you do that, does the test still pass?